pkg/controller/app_controller.go

```diff
@@ -113,6 +138,29 @@ func (a *AppController) umountFuseSidecar(pod *corev1.Pod, fuseContainer corev1.
 	}
 	return
 }
+
+func (a *AppController) killFuseProcesss(pod *corev1.Pod) error {
+	for _, cn := range pod.Spec.Containers {
+		if strings.Contains(cn.Name, config.MountContainerName) {
+			if e := a.killFuseProcess(pod, cn); e != nil {
+				return e
+			}
+		}
+	}
+	return nil
+}
+
+func (a AppController) killFuseProcess(pod *corev1.Pod, fuseContainer corev1.Container) error {
+	if fuseContainer.Name == "" {
+		return nil
+	}
+	log := klog.NewKlogr().WithName("app-ctrl").WithValues("pod", pod.Name, "namespace", pod.Namespace)
+	cmd := []string{"sh", "-c", "ps -x | grep mount.juicefs | grep -v grep | awk '{print $1}' | xargs kill"}
```

Comment:
parent process pid is 1? maybe use 1 instead of grep?

Reply:
see https://github.com/juicedata/juicefs-csi-driver/pull/1113#discussion_r1758256432

Reply:
pkill can make it shorter

```
root@juicefs-demo1:~# ./juicefs mount chaos-ee-test /mnt/jfs/chaos
...OK, chaos-ee-test is ready at /mnt/jfs/chaos.
root@juicefs-demo1:~# ps -ef | grep juicefs
root       78344       1 10 11:31 ?        00:00:00 ./juicefs mount chaos-ee-test /mnt/jfs/chaos
root       78352   78344 16 11:31 ?        00:00:00 /root/.juicefs/jfsmount mount chaos-ee-test /mnt/jfs/chaos
root       78365   77218  0 11:31 pts/2    00:00:00 grep --color=auto juicefs

# parent & child are all killed
root@juicefs-demo1:~# pkill jfsmount -e
jfsmount killed (pid 78344)
jfsmount killed (pid 78352)
```